package/setup.py

```diff
@@ -24,7 +24,8 @@
 with open("requirements.txt", "r", encoding="utf-8") as f:
     requires = [x.strip() for x in f if x.strip()]
 
-with open(path.join(here, "../README.md"), encoding="utf-8") as f:
+# The README.md file is being read from .github/README.md instead of from the root directory.
+with open(path.join(here, "../.github/README.md"), encoding="utf-8") as f:
```

Comment:
Or maybe move that `README.md` to `package/`? (That, in turn, helps with https://github.com/kedro-org/kedro-viz/pull/1766)

Reply:
@astrojuanlu  For github to read README.md it should be in `.github`, root, or docs directory. ref: [GitHub readme](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-readmes#about-readmes). and for npm we are using README.md from root  directory. We already have two README.md so I wanted to reuse from `.github` for PyPI.

Reply:
This is shorted as we are not moving existing README.md